frontend/src/components/authentication/ForgotPassword.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {useState, useCallback} from 'react'
+import {CssBaseline, TextField} from '@material-ui/core'
+import {Button, Avatar, Typography, Link, Paper, Grid, Box }  from '@material-ui/core'
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined'
+
+import * as gridSize from '../../constants/styles.values'
+import {SIX as SHADOW_DEPTH_SIX} from '../../constants/styles.values'
+import { emailFormatIsValid } from '../../utils/FormatVerifications'
+import {useStyles} from './SignUp.styles'
+import { AuthApiClient } from '../../api/authRequest'
+
+export default function ForgotPasswordSide(): React.ReactElement {
+
+  const classes = useStyles()
+
+  const initialState = {
```

Comment:
Please do not create object inside react function component (or render method for class component). Move this object outside of component.

Reply:
Done